components/Icons.tsx

```diff
@@ -0,0 +1,43 @@
+type Icons = {
+    [key: string]: JSX.Element;
+  };
+  
+ const Icons: Icons = {
```

Comment:
It'll be better if we use `ReactNode` and the `Record` generic in this case rather than a custom type

```tsx
import type { ReactNode } from 'react';

const Icons: Record<string, ReactNode> = {
```